<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Planets3D</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/icons/silk/world.png</normaloff>:/icons/silk/world.png</iconset>
  </property>
  <widget class="PlanetsWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Simulation"/>
    <addaction name="actionOpen_Simulation"/>
    <addaction name="actionAppend_Simulation"/>
    <addaction name="actionSave_Simulation"/>
    <addaction name="separator"/>
    <addaction name="actionTake_Screenshot"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionGrid"/>
    <addaction name="actionDraw_Paths"/>
    <addaction name="actionPlanet_Colors"/>
    <addaction name="actionHide_Planets"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuCamera">
    <property name="title">
     <string>Camera</string>
    </property>
    <addaction name="actionFollow_Selection"/>
    <addaction name="actionPlain_Average"/>
    <addaction name="actionWeighted_Average"/>
    <addaction name="actionClear_Follow"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuCamera"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="newPlanet_DockWidget">
   <property name="windowTitle">
    <string>New Planet</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="newPlanet_DockWidgetContents">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="3">
      <widget class="QLabel" name="newZ_Label">
       <property name="text">
        <string>Z</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="newY_Label">
       <property name="text">
        <string>Y</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="newX_Label">
       <property name="text">
        <string>X</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="newPosX_SpinBox">
       <property name="prefix">
        <string/>
       </property>
       <property name="suffix">
        <string/>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>-100.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="3">
      <widget class="QSpinBox" name="newMass_SpinBox">
       <property name="suffix">
        <string/>
       </property>
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>1000000</number>
       </property>
       <property name="singleStep">
        <number>50</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QDoubleSpinBox" name="newVelocityZ_SpinBox">
       <property name="suffix">
        <string/>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>-100.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="newMass_Label">
       <property name="text">
        <string>Mass</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="newVelocityX_Label">
       <property name="text">
        <string>Velocity</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QDoubleSpinBox" name="newPosZ_SpinBox">
       <property name="suffix">
        <string/>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>-100.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QDoubleSpinBox" name="newPosY_SpinBox">
       <property name="suffix">
        <string/>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>-100.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="newVelocityX_SpinBox">
       <property name="suffix">
        <string/>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>-100.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="newPos_Label">
       <property name="text">
        <string>Position</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QDoubleSpinBox" name="newVelocityY_SpinBox">
       <property name="suffix">
        <string/>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>-100.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="4">
      <widget class="QPushButton" name="createPlanet_PushButton">
       <property name="text">
        <string>Create</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/icons/silk/world_add.png</normaloff>:/icons/silk/world_add.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="speedControl_DockWidget">
   <property name="windowTitle">
    <string>Speed Control</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="speedControl_DockWidgetContents">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QDial" name="speed_Dial">
       <property name="maximum">
        <number>3200</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
       <property name="notchTarget">
        <double>50.000000000000000</double>
       </property>
       <property name="notchesVisible">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLCDNumber" name="speedDisplay_lcdNumber">
       <property name="smallDecimalPoint">
        <bool>false</bool>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Filled</enum>
       </property>
       <property name="value" stdset="0">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="PauseResume_Button">
       <property name="text">
        <string>Pause</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/icons/silk/control_pause_blue.png</normaloff>:/icons/silk/control_pause_blue.png</iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="FastForward_Button">
       <property name="text">
        <string>Fast Forward</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/icons/silk/control_fastforward_blue.png</normaloff>:/icons/silk/control_fastforward_blue.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>General Tools</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDelete"/>
   <addaction name="actionClear_Velocity"/>
   <addaction name="actionNew_Simulation"/>
   <addaction name="actionCenter_All"/>
   <addaction name="actionInteractive_Planet_Placement"/>
   <addaction name="actionInteractive_Orbital_Placement"/>
   <addaction name="actionDelete_Escapees"/>
  </widget>
  <widget class="QDockWidget" name="viewSettings_DockWidget">
   <property name="windowTitle">
    <string>View Settings</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="viewSettings_DockWidgetContents">
    <layout class="QFormLayout" name="formLayout_3">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="stepsPerFrameLabel">
       <property name="text">
        <string>Steps Per Frame</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="stepsPerFrameSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="trailLengthLabel">
       <property name="text">
        <string>Trail Length</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="trailLengthSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>4000</number>
       </property>
       <property name="value">
        <number>200</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="gridRangeLabel">
       <property name="text">
        <string>Grid Dimensions</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="gridRangeSpinBox">
       <property name="minimum">
        <number>4</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="singleStep">
        <number>4</number>
       </property>
       <property name="value">
        <number>32</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="trailRecordDistanceLabel">
       <property name="text">
        <string>Trail Record Distance</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="trailRecordDistanceDoubleSpinBox">
       <property name="minimum">
        <double>0.200000000000000</double>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="planetScaleLabel">
       <property name="text">
        <string>Planet Scale</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="planetScaleDoubleSpinBox">
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>8.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="firingSettings_DockWidget">
   <property name="windowTitle">
    <string>Firing Mode Settings</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="firingSettings_DockWidgetContents">
    <layout class="QFormLayout" name="formLayout_4">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="firingVelocityLabel">
       <property name="text">
        <string>Firing Velocity</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="firingVelocityDoubleSpinBox">
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="value">
        <double>10.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="firingMassLabel">
       <property name="text">
        <string>Firing Mass</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="firingMassSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000000</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QPushButton" name="toggleFiringModePushButton">
       <property name="text">
        <string>Toggle Firing Mode</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="randomSettings_DockWidget">
   <property name="windowTitle">
    <string>Random Planet Generator</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="randomSettings_DockWidgetContents">
    <layout class="QFormLayout" name="formLayout_5">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="randomAmountLabel">
       <property name="text">
        <string>Amount of Planets</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="randomAmountSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="randomRangeLabel">
       <property name="text">
        <string>Maximum Position</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="randomRangeDoubleSpinBox">
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>10.000000000000000</double>
       </property>
       <property name="value">
        <double>1000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="randomSpeedLabel">
       <property name="text">
        <string>Maximum Speed</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="randomSpeedDoubleSpinBox">
       <property name="maximum">
        <double>200.000000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="randomMassLabel">
       <property name="text">
        <string>Maximum Mass</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="randomMassDoubleSpinBox">
       <property name="minimum">
        <double>10.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>10.000000000000000</double>
       </property>
       <property name="value">
        <double>200.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QPushButton" name="generateRandomPushButton">
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="camerToolBar">
   <property name="windowTitle">
    <string>Camera Tools</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFollow_Selection"/>
   <addaction name="actionPlain_Average"/>
   <addaction name="actionWeighted_Average"/>
   <addaction name="actionClear_Follow"/>
  </widget>
  <widget class="QToolBar" name="viewToolBar">
   <property name="windowTitle">
    <string>View Tools</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionGrid"/>
   <addaction name="actionDraw_Paths"/>
   <addaction name="actionPlanet_Colors"/>
   <addaction name="actionHide_Planets"/>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/silk/cross.png</normaloff>:/icons/silk/cross.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionTake_Screenshot">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/silk/camera.png</normaloff>:/icons/silk/camera.png</iconset>
   </property>
   <property name="text">
    <string>Take Screenshot</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/silk/world_delete.png</normaloff>:/icons/silk/world_delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete Selected Planet</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionClear_Velocity">
   <property name="text">
    <string>Clear Velocity</string>
   </property>
   <property name="toolTip">
    <string>Clear the currently selected planet's velocity</string>
   </property>
  </action>
  <action name="actionNew_Simulation">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/silk/asterisk_yellow.png</normaloff>:/icons/silk/asterisk_yellow.png</iconset>
   </property>
   <property name="text">
    <string>New Simulation</string>
   </property>
   <property name="toolTip">
    <string>Start a new simulation with no planets</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionCenter_All">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/silk/arrow_inout.png</normaloff>:/icons/silk/arrow_inout.png</iconset>
   </property>
   <property name="text">
    <string>Center All</string>
   </property>
   <property name="toolTip">
    <string>Center all planets.</string>
   </property>
  </action>
  <action name="actionInteractive_Planet_Placement">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/silk/world_add.png</normaloff>:/icons/silk/world_add.png</iconset>
   </property>
   <property name="text">
    <string>Interactive Planet Placement</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="actionDraw_Paths">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Draw Paths</string>
   </property>
  </action>
  <action name="actionOpen_Simulation">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/silk/folder.png</normaloff>:/icons/silk/folder.png</iconset>
   </property>
   <property name="text">
    <string>Open Simulation</string>
   </property>
   <property name="toolTip">
    <string>Open Simulation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_Simulation">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/silk/disk.png</normaloff>:/icons/silk/disk.png</iconset>
   </property>
   <property name="text">
    <string>Save Simulation</string>
   </property>
   <property name="toolTip">
    <string>Save Simulation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/silk/information.png</normaloff>:/icons/silk/information.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionPlanet_Colors">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Planet Colors</string>
   </property>
  </action>
  <action name="actionInteractive_Orbital_Placement">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/silk/world_add.png</normaloff>:/icons/silk/world_add.png</iconset>
   </property>
   <property name="text">
    <string>Interactive Orbital Placement</string>
   </property>
   <property name="shortcut">
    <string>O</string>
   </property>
  </action>
  <action name="actionGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Grid</string>
   </property>
  </action>
  <action name="actionDelete_Escapees">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/silk/delete.png</normaloff>:/icons/silk/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete Escapees</string>
   </property>
   <property name="toolTip">
    <string>Deletes all planets too far from the universe center to see.</string>
   </property>
  </action>
  <action name="actionFollow_Selection">
   <property name="text">
    <string>Follow Selection</string>
   </property>
  </action>
  <action name="actionPlain_Average">
   <property name="text">
    <string>Plain Average</string>
   </property>
  </action>
  <action name="actionWeighted_Average">
   <property name="text">
    <string>Weighted Average</string>
   </property>
  </action>
  <action name="actionClear_Follow">
   <property name="text">
    <string>Clear Follow</string>
   </property>
  </action>
  <action name="actionHide_Planets">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hide Planets</string>
   </property>
  </action>
  <action name="actionAppend_Simulation">
   <property name="text">
    <string>Append Simulation</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlanetsWidget</class>
   <extends>QWidget</extends>
   <header>planetswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
